docs/section10/10-9.md

```diff
@@ -0,0 +1,65 @@
+                - Else, do nothing and move on to the next metric: `Else:`     
+            - if was_capacity_not_defined equals TRUE (if this equals False then carry on with logic, if it equals true carry out the 2 lines below and then skip to Rule Assertion): `if was_capacity_not_defined == TRUE:`
+                - Set most conservative efficiency (highest efficiency) equal to 3.6 from Table G3.5.1: `most_conservative_eff = 3.6`
+                - Check if the modeled efficiency equals the most conservative by using the RCT compare_standard_val function, Permit_More_Stringent_Value is defined globally and is TRUE OR FALSE: `capacity_not_defined_and_meets_most_conserv_eff_b = compare_standard_val(Permit_More_Stringent_Value, most_conservative_eff, modeled_efficiency_b, "greater than")`  func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the most_conservative_eff it would be accepted if Permit_More_Stringent_Value equals TRUE   
+            - Determine the capacity (Btuh) that should be used to determine the modeled baseline COPnfcooling efficiency based upon the number of floors that the hvac system serves: `cool_cap_hvac_b = total_cool_capacity_b/num_floors_hvac_b`
+            Determine the expected efficiency to be modeled based on the rules of App G and the size of the system
+            - Check if capacity is <65,000 Btuh: `if cool_cap_hvac_b < 65000: size_category_b = 65Btuh`
+            - Check else if capacity is <135,000 Btuh and greater or equal to 65,000 Btuh: `elif cool_cap_hvac_b < 135000 AND cool_cap_hvac_b >= 65000 : size_category_b = 65_135Btuh`
+            - Check else if capacity is <240,000 Btuh and greater or equal to 135,000 Btuh: `elif cool_cap_hvac_b < 240000 AND cool_cap_hvac_b >= 135000 : size_category_b = 135_240Btuh`
+            - Check else if capacity is <760,000 Btuh and greater or equal to 240,000 Btuh: `elif cool_cap_hvac_b < 760000 AND cool_cap_hvac_b >= 240000 : size_category_b = 240_760Btuh`
+            - Else capacity is >=760,000 Btuh: `Else: size_category_b = 760Btuh`
+            - Set the COPnf_full_load_expected_b variable using data_lookup: `COPnf_full_load_expected_b = data_lookup(table_G3_5_1,size_category_b)`  
+            **Rule Assertion:**
+            - Case 1: For each applicable hvac system, outcome is pass if the modeled efficiency is as expected (compare_standard_val func shall return True or False, "greater than" means that if the modeled_efficiency_b is greater than or equal to the COPnf_full_load_expected_b it would be accepted if Permit_More_Stringent_Value equals TRUE ) and was_efficiency_found equals True and the capacity was defined: `if compare_standard_val(Permit_More_Stringent_Value, COPnf_full_load_expected_b, modeled_efficiency_b, "greater than") == TRUE and was_efficiency_found == TRUE and was_capacity_not_defined == FALSE: PASS`
```

Comment:
It seems that we don't need was_efficiency_found==TRUE because if this is false than the compare-standard-val will not return TRUE.

Reply:
It shouldn't because then modeled_efficiency_b will equal Null so I will remove.